jwst/ami/utils.py

```diff
@@ -49,24 +49,32 @@ class Affine2d:
     in the image plane.
 
     The theorem states that if f(x,y) and F(u,v) are Fourier pairs, and
-    g(x,y) = f(x',y'), where::
+    g(x,y) = f(x',y'), where:
+
+    .. code-block:: text
```

Comment:
Note for future person: The code block type has to be defined explicitly because package default is set to Python, so this text was getting highlighted in a way that does not make sense.